### ISSUE TITLE
Fail to create a new stream app property form with the API

### ISSUE THREAD

**Issue by konekoya** [oharastream/ohara#2127]
I'm trying to add a new stream app property form into a newly created pipeline. But failed to do so with the following request:

API: __POST__ `v0/stream`
Requset:

```json
{
   "name":"asfdsdfasfdasdf",
   "jar":{
      "name":"ohara-streamapp.jar",
      "size":1930,
      "url":"http://10.1.13.102:12345/v0/downloadFiles/abc/ohara-streamapp.jar",
      "lastModified":1564989589000,
      "tags":{

      },
      "group":"abc"
   }
}
```

And the response
```json
{
   "code":"Unsupported API: v0/stream",
   "message":"please see link to find the available APIs. input url:v0/stream",
   "stack":"N/A",
   "apiUrl":"https://ohara.readthedocs.io/en/latest/rest_interface.html"
}
```

**Comment by konekoya** [comment]
@saivirtue Thanks for the help :)

**Comment by saivirtue** [comment]
It is because streamApp doesn't support **group** :)

**Comment by konekoya** [comment]
Thanks Sam! This will be fixed in #1777 :)
